randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -2207,8 +2207,26 @@ Extra - aabb: [743.8239, -217.36417, 29.29987, 834.11273, -144.96254, 81.37303]
   * Normal Door to Reflecting Pool Access/Door to Hall of the Elders
   * Extra - dock_index: 0
   * Extra - nonstandard: False
+  > Door to Crossway Access South
+      All of the following:
+          # https://youtu.be/C5DBRQ4gCIY
+          Space Jump Boots and Knowledge (Advanced) and L-Jump (Expert) and Standable Terrain (Intermediate)
+          Any of the following:
+              After Ghost Skip Barrier
+              All of the following:
+                  # https://youtu.be/A-A_Bk_Bk50
+                  Morph Ball and Movement (Intermediate)
+                  Any of the following:
+                      Morph Ball Bomb
+                      Boost Ball and Wall Boost (Beginner)
   > Behind Barrier
       Trivial
+  > Event - Elder Chamber Ghost Skip
+      All of the following:
+          # https://youtu.be/aDOnDt5AMLo
+          Morph Ball Bomb and Morph Ball and Space Jump Boots and Knowledge (Advanced) and L-Jump (Expert) and Movement (Advanced) and Standable Terrain (Intermediate) and Shoot Plasma Beam
+  > Event - Ghost Skip Barrier
```

Comment:
I really dislike that there's two events for the same thing. Let's do something like this instead:

```
Reflecting Pool Access → Event  - Deactivate Barrier
Reflecting Pool Access → Behind Barrier
Reflecting Pool Access → Crossway Access South ((Ghost Skip and Barrier) or (Ghost Skip and Barrier Skip))
Event  - Deactivate Barrier → Reflecting Pool Access
Event  - Deactivate Barrier → Behind Barrier
```

Basically, make it so that the only way to unlock the barrier is by standing at the door. From the door your options are:
- Ghost Skip
- Unlock the barrier and come back to the door
- Unlock the barrier and jump down into the ghost fight
- Don't unlock the barrier and jump down into the ghost fight

Reply:
This can all be done, except for the event connecting back to Reflecting Pool. If you have the Deactivate Barrier event node connect to Reflecting Pool Access, that would create an invalid path if you have the logic for the expert bomb jump to skip wave, since the Ghost trigger locks this door.